pages/common/mkcert.md

```diff
@@ -0,0 +1,24 @@
+# alias
+
+> Tool for making locally-trusted development certificates.
+> Homepage: <https://github.com/FiloSottile/mkcert>.
+
+- Install the local CA in the system trust store:
+
+`mkcert -install`
+
+- Generate certificate and private key for the given domain:
+
+`mkcert {{example.org}}`
+
+- Generate certificate and private key for the multiple domains:
```

Comment:
```suggestion
- Generate certificate and private key for multiple domains:
```

Also, I'd suggest changing "the given domain" in the other examples to "a given domain", as that sounds more natural to me.